### DISCUSSION TITLE
Polyrythms

### DISCUSSION THREAD

**Discussion by eh2k** [eh2k/squares-and-circles#87] [Ideas]
https://www.youtube.com/@project_jdm/featured

**Comment by jpnielsen** [comment]
Just linking my fav. VCV rack poly clock.

https://library.vcvrack.com/BaconMusic/PolyGnome

**Reply by jpnielsen** [reply]
and offcause Temps_Utile that I have.
(it is also Teensy 3.2 based - wich is no longer available, but the source code is.)

https://github.com/mxmxmx/temps_utile-/wiki/using-temps_utile

**Reply by eh2k** [reply]
I also have a temps_utile with the teensy 3.2, there I used the original firmware for a short time. At the moment I have another version of s&c on it, which so far partly unfinished and I haven't found the time to round it off yet. With the temps_utile you could theoretically send a midi signal via the uart pin (I think it was out-1) instead of trigger signals, which could be connected to the O_C trig-1 "midi" input to control the OC polyphonically and possibly do something along those lines. I'm thinking of some kind of bouncing balls with several voices/CV+gates.
